interactions/ext/sentry.py

```diff
@@ -91,12 +91,13 @@ def setup(
     bot: Client,
     token: str = None,
     filter: Optional[Callable[[dict[str, Any], dict[str, Any]], Optional[dict[str, Any]]]] = None,
+    sentry_args: dict[str, Any] = None,
```

Comment:
I'd prefer this be **kwargs, as proposed in the initial issue

Reply:
Yeah, this would work too.
The idea was to explicit pass arguments into the init()

Reply:
Asking people to enclose parameters in a dict doesn't have any benefit, makes the UX worse and the code less readable.